src/pybind/mgr/feedback/module.py

```diff
@@ -0,0 +1,110 @@
+
+"""
+Feedback module
+
+See doc/mgr/feedback.rst for more info.
+"""
+
+from mgr_module import CLIReadCommand, HandleCommandResult, MgrModule
+import errno
+from .services import CephTrackerClient
+from .model import Feedback
+
+
+class FeedbackModule(MgrModule):
+
+    # there are CLI commands we implement
+    @CLIReadCommand('set issue_key')
+    def set_api_key(self, key: str) -> HandleCommandResult:
+        """
+        Set Ceph Issue Tracker API key
+        """
+        try:
+            self.set_store('issue_tracker', key)
```

Comment:
Same here, I'd keep the naming consistent across layers (keep in mind that the internal store this will become as `mgr/feedback/api_key`, which is pretty self-explanatory):
```suggestion
            self.set_store('api_key', key)
```